setuptools/config.py

```diff
@@ -515,21 +515,29 @@ def _parse_packages(self, value):
         :param value:
         :rtype: list
         """
-        find_directive = 'find:'
+        find_directives = ['find:', 'find_namespace:']
+        trimmed_value = value.strip()
 
-        if not value.startswith(find_directive):
-            return self._parse_list(value)
+        if not trimmed_value in find_directives:
+          return self._parse_list(value)
```

Comment:
Bad indentation

Reply:
and there was even more where this came from...

Reply:
done